app/models/chargeback_container_project.rb

```diff
@@ -38,16 +38,20 @@ def self.build_results_for_report_ChargebackContainerProject(options)
     # Find Project by id or get all projects
     provider_id = options[:provider_id]
     id = options[:entity_id]
-    raise "must provide option :entity_id and provider_id" if id.nil? && provider_id.nil?
+    tag = options[:tag]
+    raise "must provide option :entity_id, provider_id or tag" if id.nil? && provider_id.nil? && tag.nil?
 
-    @groups = if provider_id == "all"
+    @groups = if tag.present?
```

Comment:
This if statement is increasingly tricky, as cases are overlapping.  AFAICT:
- specific `:tag` => pods from projects with this tag.
- `:provider_id` being "all" => all pods.
- specific `:provider_id` + `:entity_id` being "all" => pods from that provider.
- specific `:entity_id` => pods from that project.

consider:
- renaming `id` var to `entity_id` for easier following of interaction with other code, or to `project_id` to express its meaning.
- extracting this if to a helper function *and unit-testing it separately*.